apps/subject/management/commands/import-scholardb.py

```diff
@@ -64,22 +85,43 @@ def handle(self, *args, **options):
             print()
             return
 
+        for s in target_semesters:
+            self._import_scholardb(s[0], s[1], exclude_lecture, {
+                "host": host,
+                "port": port,
+                "user": user,
+                "password": password,
+                "encoding": encoding,
+            })
+
+    def _import_scholardb(self, target_year, target_semester, exclude_lecture, db_specification):
+        print(f"Importing scholardb for {target_year}-{target_semester}")
+
+        host = db_specification["host"]
+        port = db_specification["port"]
+        user = db_specification["user"]
+        password = db_specification["password"]
+        encoding = db_specification["encoding"]
+
+        rx_dept_code = re.compile(r"([a-zA-Z]+)(\d+)")
+        lecture_count = 0
+
         if not exclude_lecture:
             query = "SELECT * FROM view_OTL_charge WHERE lecture_year = %d AND lecture_term = %d" % (
-                next_year,
-                next_semester,
+                target_year,
+                target_semester,
             )
             professors = execute(host, port, user, password, query)
 
             query = "SELECT * FROM view_OTL_lecture WHERE lecture_year = %d AND lecture_term = %d ORDER BY dept_id" % (
-                next_year,
-                next_semester,
+                target_year,
+                target_semester,
             )
             rows = execute(host, port, user, password, query)
             departments = {}
             lectures_not_updated = set()
 
-            for lecture in Lecture.objects.filter(year=next_year, semester=next_semester):
+            for lecture in Lecture.objects.filter(year=target_year, semester=target_semester):
                 lectures_not_updated.add(lecture.id)
             # Make Staff Professor with ID 830
```

Comment:
```suggestion
            # Make Staff Professor with ID 830
            STAFF_PROFESSOR_ID = 830
```
과 같이 상수로 정의해두고 공유하면 좋겠습니다.

Reply:
2ded6558